src/usb/uf2/ghostfat.c

```diff
@@ -106,7 +106,7 @@ static FAT_BootBlock const BootBlock = {
     .FATCopies            = 2,
     .RootDirectoryEntries = (ROOT_DIR_SECTORS * 512 / 32),
     .TotalSectors16       = NUM_FAT_BLOCKS - 2,
-    .MediaDescriptor      = 0xF8,
+    .MediaDescriptor      = 0xF0,
```

Comment:
Please keep the 0xF8 value

Reply:
Well, OK.  I will update the FAT entries.  However, this will require some additional changes, as there are a couple other interdependencies.